internal/ignition/ignition.go

```diff
@@ -168,79 +133,43 @@ func (g *installerGenerator) createManifestDirectory(installerPath string, envVa
 	return nil
 }
 
-func (g *installerGenerator) generateLsoManifests() error {
-	g.log.Info("Creating LSO Manifests")
-	manifests, err := lso.Manifests(g.cluster.Cluster.OpenshiftVersion)
-	if err != nil {
-		g.log.Error("Error creating LSO manifests ", err)
-		return err
-	}
-	manifestDirPath := filepath.Join(g.workDir, "manifests")
-	for name, manifest := range manifests {
-		manifestPath := filepath.Join(manifestDirPath, name)
-		err := ioutil.WriteFile(manifestPath, []byte(manifest), 0600)
-		if err != nil {
-			g.log.Errorf("Failed to write file %s %s", manifestPath, name)
-			return err
-		}
-	}
-	return nil
-}
-func (g *installerGenerator) generateOcsManifests(ocsValidatorConfig *ocs.Config) error {
-	g.log.Info("Creating OCS Manifests")
-	manifests, err := ocs.Manifests(ocsValidatorConfig.OCSMinimalDeployment, g.cluster.OpenshiftVersion, ocsValidatorConfig.OCSDisksAvailable, len(g.cluster.Cluster.Hosts))
-	if err != nil {
-		g.log.Error("Cannot generate OCS manifests due to ", err)
-		return err
-	}
-	manifestDirPath := filepath.Join(g.workDir, "manifests")
+func writeManifests(workDir string, manifests map[string]string, log logrus.FieldLogger) error {
```

Comment:
Adding this method to the same receiver (g *installerGenerator) will spare the need to pass the workDir and the logger as a parameters to the method.